dtls.c

```diff
@@ -587,11 +639,38 @@ static const struct cipher_suite_param_t cipher_suite_params[] = {
 static const dtls_cipher_index_t last_cipher_suite_param =
     sizeof(cipher_suite_params) / sizeof(cipher_suite_param_t);
 
+/**
+ * Check, if cipher suite is contained in table.
+ *
+ * \param cipher_suites table with cipher-suites. Terminated with
+ *                      TLS_NULL_WITH_NULL_NULL.
+ * \param cipher_suite cipher suite
+ * \return 0, if not contained, != 0, if contained
+ */
+static inline uint8_t
+contains_cipher_suite(const dtls_cipher_t* cipher_suites, const dtls_cipher_t cipher_suite) {
+  while ((*cipher_suites != cipher_suite) &&
```

Comment:
This function would return 1 for TLS_NULL_WITH_NULL_NULL. Does it make sense to explicitly return 0 in this case, just to be on the safe side?

Reply:
ok